wemake_python_styleguide/violations/best_practices.py

```diff
@@ -65,8 +65,9 @@
    UnhashableTypeInHashViolation
    WrongKeywordConditionViolation
    WrongNamedKeywordViolation
-   MisRefactoredAssignmentViolation
-
+   MisrefactoredAssignmentViolation
```

Comment:
These changes are not related, please remove them.

Reply:
merged from upstream